README.md

```diff
@@ -22,11 +22,23 @@ Sameer Khan
 [Presentations](https://2019.texaslinuxfest.org/presentations.html)
 
 [Keynote: Linux State of the Union](http://www.camerontech.com/txlf_2019/keynote_state_of_union.pdf)  
-[Becoming Dangerous with Kubernetes](https://speakerdeck.com/lastcoolnameleft/kubernetes-fundamentals)  
+[Ansible Beginner to Brilliant](http://slides.unsupported.io/ansible-b2b-slides/)  
+[Using strace to know what you don’t know](https://github.com/linuxaustin/OtherSlides/blob/master/UsingStrace_Linuxfest_-_Ryan_Robson.pdf)  
+[Becoming Dangerous with Kubernetes](https://ibm.gitlab.io/workshop/cloud/)  
 [An Intro to MySQL](https://www.slideshare.net/davidmstokes/mysql-baics-texas-linxufest-beginners-tutorial-may-31st-2019)  
+[You're Worth More Than You Know, Matching your Skills to Employeers](https://github.com/linuxaustin/OtherSlides/blob/master/You_are_worth_more_extended_opt.pdf)  
+[New Features in MySQL for Developers](https://www.slideshare.net/davidmstokes/mysql-8-a-new-beginning-sunshine-phpphp-uk-updated)  
+[Kids and Technology: Using OpenSource to Guide Learning](https://github.com/linuxaustin/OtherSlides/blob/master/children_and_technology.pdf)  
 [Continuous Kernel Integration](https://docs.google.com/presentation/d/1T0JaRA0wtDU0aTWTyASwwy_ugtzjUcw_ZDmC5KFzw-A)  
-[I was too burned out to name this talk](https://docs.google.com/presentation/d/1uYY8Ezw8rMhMjRA_d-H8rnv4AWqROyBx86HTcGFEtdQ)  
```

Comment:
This one appears to have been dropped out of the list.  I'm guessing that was an accident and will re-add.  Let me know if there was some reason to remove it.

Reply:
Never mind.  I see now this was listed on the conference site with the Title "Understanding Burnout", so just renamed not removed.